### PR TITLE
location_philmel: convert to new location file and add new accesspoints

### PR DESCRIPTION
**Pull request by Petrella**
Philmel needed some adjustements due to adding some access points with special config and ssid names

### CODE REVIEW

File: group_vars/all/wireless_profiles.yml
Comment (Noki) [inline group_vars/all/wireless_profiles.yml:1]: This file should not be changed. Wireless_profiles moved to the locations and should not be added back here.

**Review by Noki — CHANGES_REQUESTED** [review]
When switching to the single-file-format, the definitions needs to be moved. Please delete the files in group_vars and host_vars so they only exist in locations/philmel.yml from now on.

**Comment by FFHener** [comment]
Please mark the pr as a draft until its ready for a review

**Comment by FFHener** [comment]
Closing as there is a new version of this PR

**Pull request closed without merging**